components/TopicFootballPage.jsx

```diff
@@ -0,0 +1,10 @@
+const TopicFootballPage = () => {
+  return (
+    <div>
```

Comment:
The only problem having it like this is that if another topic was added you would have to change this